lib/connect-session-sequelize.js

```diff
@@ -68,16 +74,32 @@ module.exports = function SequelizeSessionInit (Store) {
           }
           debug('FOUND %s with data %s', session.sid, session.data)
 
+          if (self.options.secret) {
+            self.options.kruptein.get(self.options.secret, session.data, function (err, plaintext) {
+              if (err) return null
+
+              session.data = JSON.parse(plaintext)
+            })
+          }
+
           return JSON.parse(session.data)
         })
         .asCallback(fn)
     }
 
     set (sid, data, fn) {
       debug('INSERT "%s"', sid)
-      const stringData = JSON.stringify(data)
+      let stringData = JSON.stringify(data)
       const expires = this.expiration(data)
 
+      if (this.options.secret) {
+        this.options.kruptein.set(this.options.secret, stringData, function (err, ciphertext) {
+          if (err) return
+
+          stringData = ciphertext
```

Comment:
that won't work either. You need to wrap everything within that callback otherwise it won't work.